va_explorer/va_data_management/management/commands/run_coding_algorithms.py

```diff
@@ -1,15 +1,47 @@
+import argparse
+import pandas as pd
+import time
 from django.core.management.base import BaseCommand
+from django.db.models import F
 from va_explorer.va_data_management.utils.coding import run_coding_algorithms
+from va_explorer.va_data_management.models import VerbalAutopsy, CauseOfDeath
 
 
```

Comment:
@babraham33 Can you add the algorithm settings here and first validate that the values the user has set in the .env file are valid? If the user-supplied values in the .env are invalid, either fallback to the default setting or exit the program. 

Example: In the .env file, I set `INTERVA_HIV=t` (invalid). But, that wasn't hit until the cause of death table had already been cleared. We probably want the program to exit if an invalid setting is passed in before clearing the previous causes of death.

Reply:
Great point. Will do!